src/pdf_beamtime/src/pdf_beamtime_server.cpp

```diff
@@ -8,13 +8,32 @@ PdfBeamtimeServer::PdfBeamtimeServer(
   const std::string & move_group_name = "ur_manipulator",
   const rclcpp::NodeOptions & options = rclcpp::NodeOptions(),
   std::string action_name = "pdf_beamtime_action_server")
-: node_(std::make_shared<rclcpp::Node>("simple_server", options)),
+: node_(std::make_shared<rclcpp::Node>("pdf_beamtime_server", options)),
   move_group_interface_(node_, move_group_name),
   planning_scene_interface_()
 {
   // Add the obstacles
   planning_scene_interface_.applyCollisionObjects(create_env());
 
+  // Create the services
+  env_refresh_service_ = node_->create_service<NewObstacleMsg>(
+    "pdf_new_obstacle",
+    std::bind(
+      &PdfBeamtimeServer::new_obstacle_service_cb, this, std::placeholders::_1,
+      std::placeholders::_2));
+
+  update_obstacles_service_ = node_->create_service<UpdateObstaclesMsg>(
+    "pdf_update_obstacles",
+    std::bind(
+      &PdfBeamtimeServer::update_obstacles_service_cb, this, std::placeholders::_1,
+      std::placeholders::_2));
+
+  remove_obstacles_service_ = node_->create_service<UpdateObstaclesMsg>(
+    "pdf_remove_obstacle",
+    std::bind(
+      &PdfBeamtimeServer::remove_obstacles_service_cb, this, std::placeholders::_1,
+      std::placeholders::_2));
+
```

Comment:
I feel like these should be 3 different message types, but I'm open to chat it through.

Reply:
New message DeleteObstacleMsg.srv is added.
https://github.com/maffettone/erobs/blob/ed89fd89b78dfd1d51e227f933d04c9d4588b283/src/pdf_beamtime_interfaces/srv/DeleteObstacleMsg.srv#L1-3